common-util/Details/ServiceState/2StepActiveRegistration/index.jsx

```diff
@@ -55,7 +76,10 @@ const ActiveRegistration = ({
         isDisabled={btnProps.disabled}
       />
       <Text type="secondary">
-        {`Adding instances will cause a bond of ${totalBond || '--'} ETH`}
+        {`Adding instances will cause a bond of ${totalBondEthToken} ETH`}
```

Comment:
What does "cause a bond" mean – the user needs to pay these amounts in ETH and TKN, no? So rather than `will cause`, `requires` makes more sense.